### ISSUE TITLE
NVDA doesn't read the Windows start menu properly, or whatsapp

### ISSUE THREAD

**Issue by just-anotheruser** [nvaccess/nvda#17674]
### Detailed description of the issue
For some weird reason, when I woke up this morning and opened the start menu, nvda was reading just the first few letters of the word then it stopped..
If i arrowed down the same thing was happening, just the first few letters were read then nothing.
The same problem for whatsapp, if i went to the application it was reading just 
the beginning of the word.
For example:
Let's say that the names that it should read are:
loren,
Sarah,
Kaytlin
It reads just:
Lor
Sar
Ka.
But the issue doesn't appear only when i read the list of contacts,
it appears when i try to message them, letters don't get read, the whole application from settings to the list has the same issue.
It doesn't happen all the time, but sometimes I get the same thing when I try to look at windows settings too.

### Why are the other templates not appropriate in this case?
Because I asked other friends who use the same version that I use and they don't have this issue.

### Have you asked for advice on how to report this issue via a community discussion? If so, please link to the discussion
no.

### Steps to reproduce or illustrate the issue (if applicable)
unknown

### Expected outcome or behaviour (if applicable)
NVDA doesn't read the windows start menu properly.

### System configuration or software environment (if applicable)
Windows 11 latest version

### Additional information (if applicable)
I tried to restart NVDA without addons,
Ran the registration tool,
Uninstalled and Removed NVDA from appdata,
Reinstalled it, tried to run a portable version,
did a system restore point and after all these steps I still have the same issue.

[nvda log 2025 addons disabled.txt](https://github.com/user-attachments/files/18662525/nvda.log.2025.addons.disabled.txt)

**Comment by Emil-18** [comment]
@just-anotheruser Could you upload the log while the loggin level is set to debug? To do this, press NVDA +q to exit NVDA and select "Restart with debug logging enabled". Make sure "Show exit options when exiting NVDA", found in general settings, is checked.

**Comment by seanbudd** [comment]
Welcome @just-anotheruser ,

We are unable to process this issue as it stands.

This is a bug report. Please open a new issue and use the bug template and fill out the issue template fully.


Kind Regards,
NV Access Software Developers
